tfkbnufft not working in graph mode for tf 2.2

With tf 2.2 in tests I get the following error:

```
E         ValueError: in user code:
E         
E             /home/zaccharie/workspace/tfkbnufft/tfkbnufft/nufft/interp_functions.py:211 kbinterp  *
E                 for b in tf.range(tf.shape(x)[0]):
E             /home/zaccharie/workspace/fastmri-reproducible-benchmark/venv/lib/python3.6/site-packages/tensorflow/python/autograph/operators/control_flow.py:344 for_stmt
E                 iter_, extra_test, body, get_state, set_state, symbol_names, opts)
E             /home/zaccharie/workspace/fastmri-reproducible-benchmark/venv/lib/python3.6/site-packages/tensorflow/python/autograph/operators/control_flow.py:532 _tf_range_for_stmt
E                 opts)
E             /home/zaccharie/workspace/fastmri-reproducible-benchmark/venv/lib/python3.6/site-packages/tensorflow/python/autograph/operators/control_flow.py:862 _tf_while_stmt
E                 _verify_loop_init_vars(init_vars, symbol_names)
E             /home/zaccharie/workspace/fastmri-reproducible-benchmark/venv/lib/python3.6/site-packages/tensorflow/python/autograph/operators/control_flow.py:111 _verify_loop_init_vars
E                 raise ValueError('"{}" may not be None before the loop.'.format(name))
E         
E             ValueError: "params[dims]" may not be None before the loop.
```

This doesn't happen in tf 2.1 and doesn't happen when eager execution for tf.functions is turned on.

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
-tensorflow==2.1.0
+tensorflow>=2.1.0
 numpy
 scipy

--- a/tfkbnufft/__init__.py
+++ b/tfkbnufft/__init__.py
@@ -1,6 +1,6 @@
 """Package info"""
 
-__version__ = '0.0.2'
+__version__ = '0.0.3'
 __author__ = 'Zaccharie Ramzi'
 __author_email__ = 'zaccharie.ramzi@inria.fr'
 __license__ = 'MIT'

--- a/tfkbnufft/nufft/interp_functions.py
+++ b/tfkbnufft/nufft/interp_functions.py
@@ -208,8 +208,8 @@ def kbinterp(x, om, interpob):
     )
     # run the table interpolator for each batch element
     # TODO: look into how to use tf.while_loop
+    params['dims'] = tf.cast(tf.shape(x[0])[1:], 'int64')
     for b in tf.range(tf.shape(x)[0]):
-        params['dims'] = tf.cast(tf.shape(x[b])[1:], 'int64')
         # tm are the localized frequency locations
         # view(x.shape[1], 2, -1) allows to have the values of each point
         # on the grid in a list, (x.shape[1] is the number of coils and 2
@@ -283,11 +283,11 @@ def adjkbinterp(y, om, interpob):
     )
     # run the table interpolator for each batch element
     # TODO: look into how to use tf.while_loop
+    params['dims'] = tf.cast(grid_size, 'int64')
     for b in tf.range(tf.shape(y)[0]):
         # phase for fftshift
         y_shifted = y[b] * tf.math.conj(tf.exp(1j * tf.cast(tf.linalg.matvec(tf.transpose(om[b]), n_shift), y[b].dtype))[None, ...])
 
-        params['dims'] = tf.cast(grid_size, 'int64')
 
         x = x.write(b, run_interp_back(y_shifted, tm[b], params))
 

